package.json

```diff
@@ -191,7 +191,8 @@
       "node-menu",
       "metrics-cluster-feature",
       "license-menu-item",
-      "kube-object-event-status"
+      "kube-object-event-status",
+      "example-extension"
```

Comment:
`example-extension` should NOT be included in PR (used only internally / for demo)

Reply:
good catch :)